lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -518,24 +516,38 @@ def generate_test_result_tree(tmp_path: Path, dir_name: str) -> Path:
                 parents=True, exist_ok=True
             )
             (sub_dir / "subdir1" / "subdir14" / "subdir141" / "f1411.txt").touch()
-            sym_file = sub_dir / "subdir1" / "f12_sym"
-            os.symlink(Path(".."), sym_file)
-            sym_file = sub_dir / "subdir1" / "subdir12" / "f121_sym"
-            os.symlink(Path("../..") / "subdir1" / "subdir15", sym_file)
-            sym_file = sub_dir / "subdir1" / "subdir12" / "f122_sym"
-            os.symlink(Path(".") / "bad_subdir" / "nonexistent_file.txt", sym_file)
-            sym_file = sub_dir / "subdir1" / "subdir13" / "f131_sym"
-            os.symlink(Path("/etc") / "passwd", sym_file)
-            sym_file = sub_dir / "subdir1" / "subdir14" / "subdir141" / "f1412_sym"
-            os.symlink(sub_dir / "subdir1" / "f11.txt", sym_file)
-            sym_file = sub_dir / "subdir1" / "subdir14" / "subdir141" / "f1413_sym"
-            os.symlink(Path("..") / "subdir141", sym_file)
-            sym_file = sub_dir / "subdir1" / "subdir14" / "subdir141" / "f1414_sym"
-            os.symlink(Path(".") / "f1411.txt", sym_file)
-            sym_file = sub_dir / "subdir1" / "subdir14" / "subdir141" / "f1415_sym"
-            os.symlink(Path(".") / "f1412_sym", sym_file)
-            sym_file = sub_dir / "subdir1" / "subdir14" / "subdir141" / "f1416_sym"
-            os.symlink(Path("../..") / "subdir12" / "f122_sym", sym_file)
+            os.symlink(Path("../../.."), sub_dir / "subdir1" / "f12_sym")
+            os.symlink(
+                Path("../..") / "subdir1" / "subdir15",
```

Comment:
Since `subdir15` is not expected to exist (it's not...right?), maybe we should give it a more expressive name?  (Like "nonexistent_dir"?)

Reply:
I struggled with the naming conventions here. Riya chose to give hierarchical names, which are useful in one sense, but I had to keep coming back here to figure out which were good/bad/etc. I did consider changing the names to record the function rather than just the hierarchy, and I just didn't want to get into making sure I'd tracked everything down.

Reply:
Yeah, let's not track them all down, but we now know about this one....  😁